confgenerator/confgenerator.go

```diff
@@ -379,9 +380,14 @@ func (l *Logging) generateFluentbitComponents(ctx context.Context, userAgent str
 		BufferInMemory: true,
 	}.Components(ctx, fluentBitSelfLogTag)...)
 
+	out = append(out, LoggingReceiverFilesMixin{
+		IncludePaths:   []string{"${logs_dir}/../health-checks.log"},
```

Comment:
I think this path will fail (or maybe not) on Windows, since `health-checks.log` shows in the top level log folder for each platform, so the `../` may fail.

[1] https://cloud.google.com/logging/docs/agent/ops-agent/troubleshoot-find-info#health-checks

Reply:
Fixed!